extension/packagemanagerextension/serverextension/projectmanagement.py

```diff
@@ -0,0 +1,187 @@
+import json
+import os
+
+from notebook.base.handlers import (
+    APIHandler,
+    json_errors,
+)
+from tornado import web, escape
+
+from .envmanager import EnvManager, package_map
+
+from os.path import expanduser
+home = expanduser("~")
+
+def relativeDir(directory):
```

Comment:
Repeated code

Reply:
I've now put it under ProcessHelper class for ease of reusing.